tests/trestle/tasks/xccdf_result_to_oscal_ar_test.py

```diff
@@ -0,0 +1,444 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""XCCDF to OSCAL task tests."""
+
+import configparser
+import os
+import pathlib
+import uuid
+
+from _pytest.monkeypatch import MonkeyPatch
+
+from tests.test_utils import text_files_equal
+
+import trestle.tasks.xccdf_result_to_oscal_ar as xccdf_result_to_oscal_ar
+import trestle.transforms.implementations.xccdf as xccdf
+from trestle.tasks.base_task import TaskOutcome
+
+
+class MonkeyBusiness():
+    """Monkey business."""
+
+    def uuid_mock1(self):
+        """Mock uuid v1."""
+        return uuid.UUID('56666738-0f9a-4e38-9aac-c0fad00a5821')
+
+    def uuid_mock2(self):
+        """Mock uuid v2."""
+        return uuid.UUID('46aADFAC-A1fd-4Cf0-a6aA-d1AfAb3e0d3e')
+
+
+cf01 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar.config'
+cf02 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-compressed.config'
+cf03 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-fetcher.config'
+cf04 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-bad-yaml.config'
+cf05 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-1.3.5.config'
+cf06 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-xml-rhel7.config'
+cf07 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-xml-ocp4.config'
+cf08 = 'tests/data/tasks/xccdf/test-xccdf-result-to-oscal-ar-configmaps.config'
+
+
+def setup_config(path: str):
+    """Config."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(path)
+    config.read(config_path)
+    return config
+
+
+def test_xccdf_print_info(tmp_path):
+    """Test print_info call."""
+    config = setup_config(cf01)
+    section = config['task.xccdf-result-to-oscal-ar']
+    section['output-dir'] = str(tmp_path)
+    tgt = xccdf_result_to_oscal_ar.XccdfResultToOscalAR(section)
+    retval = tgt.print_info()
+    assert retval is None
+
```

Comment:
Here's the thing: this is a rework of trestle task `osco-result-to-oscal-ar` to change its name to `xccdf-result-to-oscal-ar`. As much as possible I re-used existing data and code. I could upgrade the test cases here as you suggest, but that seems like a different issue than renaming the task. Eventually `osco-result-to-oscal-ar` should be removed, but that can be saved for the next breaking release. If you agree, I can open another PR to improve existing `task` test cases, since many already have this same issue.

Reply:
Okay, in such case we can leave it until the next time.